datalad/cmdline/main.py

```diff
@@ -557,6 +560,12 @@ def main(args=None):
                 # had no code defined
                 sys.exit(exc.code if exc.code is not None else 1)
             except Exception as exc:
+                title = "DataLad Cmd Error: %s" % exc.__class__.__name__
```

Comment:
I think we can strip this coming prefix (no unique info), add a new label eg helpme or autofiled, and include instead limited in length str(exc) .

Reply:
Fixed locally.